src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="serch"
+        class="search_svg"
+      >
+
+      <input
+        type="text"
+        name="searchBar"
+        class="search search--large"
```

Comment:
you can apply BEM methodology for naming classes
```suggestion
        class="form__input form__input--large"
```